.gitignore

```diff
@@ -42,6 +42,7 @@ doc/*/Makefile
 extra/Makefile
 extra/*/Makefile
 perf/Makefile
+perf/lua/Makefile
```

Comment:
The PR title says `[DRAFT] <...>`, but you asked me for review. Aren't you forgot to change the title?

Reply:
For sure! Thanks! Updated the title.